less-builder.js

```diff
@@ -23,7 +23,7 @@ exports.bundle = function (loads, compileOpts, outputOpts) {
 	var _this = this;
 
 	var stubDefines = loads.map(function (load) {
-		return "System\.register('" + load.name + "', [], false, function() {});";
+		return (compileOpts.systemGlobal || 'System') "\.register('" + load.name + "', [], false, function() {});";
```

Comment:
I believe you're missing a `+` operator to concatenate the output.

```js
return (compileOpts.systemGlobal || 'System') + "\.register('"
```

because the return should be a string.